src/api/cvc4cpp.h

```diff
@@ -0,0 +1,2372 @@
+     * @return the Boolean constant
+     * @param val the value of the constant
+     */
+    Term mkBoolean(bool val) const;
+
+    /**
+     * Create an Integer constant.
+     * @param s the string represetntation of the constant
+     * @param base the base of the string representation
+     * @return the Integer constant
+     */
+    Term mkInteger(const char* s, uint32_t base = 10) const;
+
+    /**
+     * Create an Integer constant.
+     * @param s the string represetntation of the constant
+     * @param base the base of the string representation
+     * @return the Integer constant
+     */
+    Term mkInteger(const std::string& s, uint32_t base = 10) const;
+
+    /**
+     * Create an Integer constant.
+     * @param val the value of the constant
+     * @return the Integer constant
+     */
+    Term mkInteger(int32_t val) const;
+
+    /**
+     * Create an Integer constant.
+     * @param val the value of the constant
+     * @return the Integer constant
+     */
+    Term mkInteger(uint32_t val) const;
+
+    /**
+     * Create an Integer constant.
+     * @param val the value of the constant
+     * @return the Integer constant
+     */
+    Term mkInteger(int64_t val) const;
+
+    /**
+     * Create an Integer constant.
+     * @param val the value of the constant
+     * @return the Integer constant
+     */
+    Term mkInteger(uint64_t val) const;
+
+    /**
+     * Create a constant representing the number Pi.
+     * @return a constant representing Pi
+     */
+    Term mkPi() const;
+
+    /**
+     * Create an Real constant.
+     * @param s the string represetntation of the constant
+     * @param base the base of the string representation
+     * @return the Real constant
+     */
+    Term mkReal(const char* s, uint32_t base = 10) const;
+
+    /**
+     * Create an Real constant.
+     * @param s the string represetntation of the constant
+     * @param base the base of the string representation
+     * @return the Real constant
+     */
+    Term mkReal(const std::string& s, uint32_t base = 10) const;
+
+    /**
+     * Create an Real constant.
+     * @param val the value of the constant
+     * @return the Real constant
+     */
+    Term mkReal(int32_t val) const;
+
+    /**
+     * Create an Real constant.
+     * @param val the value of the constant
+     * @return the Real constant
+     */
+    Term mkReal(int64_t val) const;
+
+    /**
+     * Create an Real constant.
+     * @param val the value of the constant
+     * @return the Real constant
+     */
+    Term mkReal(uint32_t val) const;
+
+    /**
+     * Create an Real constant.
+     * @param val the value of the constant
+     * @return the Real constant
+     */
+    Term mkReal(uint64_t val) const;
+
+    /**
+     * Create an Rational constant.
+     * @param num the value of the numerator
+     * @param den the value of the denominator
+     * @return the Rational constant
+     */
+    Term mkRational(int32_t num, int32_t den) const;
```

Comment:
As discussed: mkRational -> mkReal (rationals are reals and SMT-LIB only talks about reals).